guides/typography.md

```diff
@@ -55,6 +55,18 @@ descendant native elements.
 </section>
 ```
 
+When applying the `mat-typography` CSS class, the descendent element rule will by default have
+higher specificity than the rules based on the CSS classes above.  Therefore, if you would like to
+be able to override a `mat-typography` descendent element rule using one of the CSS classes, you'll
+need to generate rules for the CSS classes as descendents so they'll have higher specificity than
+the descendent element rules like so:
+
+```scss
+.mat-typography {
+  @include mat-base-typography($typography-config, ".mat-typography-tags-override-classes");

```

Comment:
Should use single quotes. Also since this is an example, can it use a more concise class name?

Reply:
Cool beans.  If you folks agree this documentation is even worthwhile I'll change it :).